index.js

```diff
@@ -213,14 +213,26 @@ async function installBundler(bundlerVersionInput, platform, rubyPrefix, engine,
 }
 
 async function bundleInstall(platform, engine, version) {
-  if (!fs.existsSync('Gemfile')) {
-    console.log('No Gemfile, skipping "bundle install" and caching')
-    return
+  if (await bundleInstallSpecific('gems.rb', 'gems.locked', platform, engine, version)) {
```

Comment:
`bundleInstallSpecific` does not `return` in the successful case.
Does the `if` work here? It doesn't seem so:
```
> async function foo() { console.log("foo"); }
> async function call() { var r=await foo(); console.log(r); if (r) { console.log("OK") } else { console.log("NOPE") } }
undefined
> call()
foo
Promise { <pending> }
> undefined
NOPE
```